lib/utils.js

```diff
@@ -27,6 +27,15 @@ function renderPath (path, res, next) {
   })
 }
 
+// exports.addNunjucksFilters = function (env) {
```

Comment:
Perhaps this file should be removed from the changes? We can't merge commented out code. (validator.js is an empty, seemingly unused file that can also be removed)